Group1/src/main/java/com/stuypulse/robot/constants/Settings.java

```diff
@@ -28,4 +30,21 @@ interface FF {
             SmartNumber kA = new SmartNumber("Drivetrain/kA", 0);
         }
     }
+
+    public interface Intake {
+        SmartAngle RETRACT_ANGLE = new SmartAngle("Intake/Retract Angle", Angle.fromDegrees(0));
+        SmartAngle EXTEND_ANGLE = new SmartAngle("Intake/Extend Angle", Angle.fromDegrees(90));
```

Comment:
Find extend angle?

Reply:
doesnt really matter -- as long as it works in simulation